src/isomorphic/React.js

```diff
@@ -13,7 +13,7 @@
 
 var ReactBaseClasses = require('ReactBaseClasses');
 var ReactChildren = require('ReactChildren');
-var ReactDOMFactories = require('ReactDOMFactories');
+var reactDOMFactories = require('react-dom-factories');
```

Comment:
sooo I'm concerned about this circular dependency (this imports react-dom-factories imports react which is this) -- I *think* it's okay if we put it in a getter that requires it asynchronously which we'd need anyway in order to warn for it.